### PR TITLE
Text updates

### PR DESCRIPTION
**Pull request by mikeauclair**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@ Ad Hoc homework problems
 ========================
 
 Each directory contains a homework problem, four in total. Please complete one--or
-more--of the four before submitting. Make sure to read the README in each directory,
+more--of the problems before submitting. Make sure to read the README in each directory,
 it will give you important information on completing each problem.
 
 This is an open-book test, so Google away, but please don't copy your work from pre-existing solutions you may find or ask
-others to help you complete the exercises.
+others to help you complete the exercises. Solutions should be able to be run on a UNIX command line (Linux/OS X). We're quite familiar with Go, Ruby, Javascript, and Python, but if your submission is in some other language (which is OK), please let us know how we can set up our environment to run your code.
```
Comment (earthboundkid) [inline README.md]: OS X has changed its name to macOS.

File: README.md [hunk 2]
```diff
@@ -37,10 +37,7 @@ To submit your homework, zip up the contents of your git repo in a .zip file.
 Please remove the .git directory, and *anything else that might identify you*.
 If you want to include any comments with your submission, add them to the
 COMMENTS file. If there's anything important for us to know about how to run
-your code, please include that information in the COMMENTS file. We're quite familiar with Go,
-Ruby, Javascript, and Python, but if your submission is in some other
-language (which is OK), please let us know how we can set up our environment
-to run your code. Feel free to include any other information relevant to your
+your code, please include that information in the COMMENTS file. Feel free to include any other information relevant to your
```
Comment (earthboundkid) [inline README.md:39]: We should ask them to remember to stay anonymous. Some COMMENTS files say the person's name.

**Comment by earthboundkid** [comment]
Can we have So Company look at our readme? :-)

File: README.md (hunk 2 above)
Comment (mikeauclair) [reply README.md:39]: We do, 3 lines up.

File: README.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Well, that depends on the position you're applying for! Here are some suggestion
  - DevOps Engineer (Site Reliability): [provision](https://github.com/adhocteam/homework/tree/master/provision) and [proto](https://github.com/adhocteam/homework/tree/master/proto)
  - Web Designer with Accessibility / 508 Expertise: [accessibility](https://github.com/adhocteam/homework/tree/master/accessibility)
 
-These are just suggestions; if you really want to give all four problems a shot, go for it!
+These are just suggestions; if you really want to give other problems a shot, go for it! Just be cognizant of the fact that humans need to grade these when choosing how many problems to try.
```
Comment (aub) [inline README.md:20]: I'm not sure what you're going for here. Trying to get them to do fewer exercises? If that's the case, I'd prefer to be more explicit about it with something like "we'd prefer that you start with one or two exercises, and may ask for more if those go well". I guess I don't get why we would tell them to do fewer though.

Comment (mikeauclair) [reply README.md:20]: We just got our first 5-homework submission, it's untenable.

Comment (mikeauclair) [reply README.md:20]: I would love to tell people "Don't do more than 3 unless asked" TBH

Comment (aub) [reply README.md:20]: Ok yeah, any reason not to just be explicit about it then? "Please don't submit more than three exercises to start" or something like that?

Comment (mikeauclair) [reply README.md:20]: I can't think of any reason, just figured I'd throw out the soft-sell to start

**Pull request merged**